src/data/perf_stat.rs

```diff
@@ -140,6 +141,16 @@ impl CollectData for PerfStatRaw {
                         "Intel(R) Xeon(R) Platinum 8488C" => SPR_CTRS.to_vec(),
                         _ => Vec::new(),
                     };
+                } else if cpu_info.vendor == "AuthenticAMD" {
+                    warn!("Event multiplexing may result in bad PMU data."); //TODO: mitigate bad PMU data on AMD instances
+                    perf_list = amd_perf_events::PERF_LIST.to_vec();
+
+                    /* Get Model specific events */
+                    platform_specific_counter = match cpu_info.model_name.as_str() {
+                        "AMD EPYC 9R14" => GENOA_CTRS.to_vec(),
+                        "AMD EPYC 7R13 Processor" => MILAN_CTRS.to_vec(),
```

Comment:
Some processors may have different model name. For example, ""AMD EPYC 7R13 48-Core Processor" on c6a metal instances.

Reply:
Are you saying that because you're planning on making changes here to do, for example, a substring or prefix match? Or are you saying it because you want to know whether it matters if metal works?

Reply:
Just wanted to note it here. Have a prefix match for the first 13 characters and will push changes soon.